example/src/Keyboard/KeyboardPage.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Keyboard, TextInput } from 'smartway-react-native-ui';
+
+export const KeyboardPage = () => {
+    const [disabled, setDisabled] = useState<boolean>(false);
+    const [upperInputValue, setUpperValue] = useState<string>('');
+    const [upperInputFocused, setUpperFocused] = useState(false);
+    const [bottomInputFocused, setBottomFocused] = useState(false);
+    const [bottomInputValue, setBottomValue] = useState<string>('');
+
+    useEffect(() => {
+        if (
+            (upperInputFocused && upperInputValue.length === 1) ||
+            (bottomInputFocused && bottomInputValue.length === 1)
+        ) {
+            setDisabled(true);
+        } else {
+            setDisabled(false);
+        }
+    }, [upperInputFocused, bottomInputFocused, upperInputValue, bottomInputValue]);
+
+    const styles = StyleSheet.create({
+        container: {
+            flex: 1,
+            padding: 16,
+        },
+    });
+
+    return (
+        <>
+            <View style={styles.container}>
+                <TextInput
+                    showSoftInputOnFocus={false}
+                    textType="information"
+                    value={upperInputValue}
+                    onChangeText={setUpperValue}
+                    style={{ width: 60 }}
+                    maxLength={1}
+                    isFocused={setUpperFocused}
+                />
+                <TextInput
+                    showSoftInputOnFocus={false}
+                    textType="information"
+                    value={bottomInputValue}
+                    onChangeText={setBottomValue}
+                    style={{ width: 60 }}
+                    maxLength={1}
+                    isFocused={setBottomFocused}
+                />
+            </View>
+            <Keyboard
```

Comment:
Can we auto detect filed that has focus and fill it ?
How it works if we have 10 inputs to fill ?

Reply:
good point, thanks

Reply:
I may have answered this here https://github.com/ZeroGachis/smartway-react-native-ui/pull/31#discussion_r1142660455